Modules/posixmodule.c

```diff
@@ -15134,11 +15135,12 @@ os_DirEntry_inode_impl(DirEntry *self)
         }
 
         self->win32_file_index = stat.st_ino;
+        self->win32_file_index_high = stat.st_ino_high;
         self->got_file_index = 1;
     }
     static_assert(sizeof(unsigned long long) >= sizeof(self->win32_file_index),
                   "DirEntry.win32_file_index is larger than unsigned long long");
```

Comment:
This assertion isn't meaningful. We control the type of the field. Rather than adding a check for the new field, let's just remove it.